student.rb

```diff
@@ -3,7 +3,12 @@
 class Student < Person
   def initialize(*args, classroom:, **kwargs)
     super(*args, **kwargs)
+    self.classroom = classroom
+  end
+
+  def classroom=(classroom)
     @classroom = classroom
+    @classroom.students.push(self) unless @classroom.students.include?(self)
   end
```

Comment:
- [ ] good job setting up the association 👍 however the `classroom` should have an attr_reader as well.

Reply:
Hello, thanks. I've made this change.